.github/ISSUE_TEMPLATE/create-new-section-template.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Create a new section
+about: For issues requesting for the need of an entirely new section(sidebars) in the documentation
```

Comment:
This would sound better as "For documentation issues requiring an entirely new section in the sidebar"